_posts/en/newsletters/2020-03-25-newsletter.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Bitcoin Optech Newsletter #90'
+permalink: /en/newsletters/2020/03/25/
+name: 2020-03-25-newsletter
+slug: 2020-03-25-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes several questions and answers from the
+Bitcoin StackExchange and describes notable changes to popular Bitcoin
+infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+*No significant news about Bitcoin infrastructure development this week.*
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Why does the banscore default to 100?]({{bse}}93795) User Anonymous
+describes some history behind `banscore`, which protects nodes
+from misbehaving peers. Although some offenses result in a 100
+point increase---and thus the immediately banning the offending peer under the default
+`banscore` setting---other offenses detailed in
+[`net_processing.cpp`][bitcoin net processing] have different scores.
+
+- [Why is block 620826's timestamp 1 second before block 620825?]({{bse}}93696)
+Andrew Chow and Raghav Sood clarify that a block header's timestamp field is not
+required to have a greater value than previous blocks. The requirements are
+instead that a new block's timestamp must be greater than the median
+timestamp of the past 11 blocks but not more than two hours into the
+future.
```

Comment:
I think this could say something like '2 hours in the future from when the block was received' or similar (as it stands it could be read as '2 hours into the future from the median time of the past 11 blocks'